starcheck/src/starcheck.pl

```diff
@@ -674,7 +675,7 @@ sub json_obsids{
     $out .= sprintf "<A HREF=\"#obsid$obs{$obsid}->{obsid}\">OBSID = %5s</A>", $obs{$obsid}->{obsid};
     $out .= sprintf " at $obs{$obsid}->{date}   ";
 
-    my $guide_count = $obs{$obsid}->count_guide_stars();
+    my $guide_count = $obs{$obsid}->{figure_of_merit}->{guide_count};
```

Comment:
Here, by setting guide count from the value stored in the structure we save a call for the place where we were previously checking guide count up in the summary.